src/components/startscreen/StartScreen.module.scss

```diff
@@ -1,14 +1,48 @@
 .StartScreen {
   position: absolute;
-  left: 0.5rem;
-  top: 0.5rem;
-  width: 47.7rem;
+  left: 0;
+  top: 0;
   height: 100%;
+  width: 100%;
   background: rgba(0, 0, 0, 0.4);
+  justify-items: center;
+  background-color: #003459;
+  text-align: center;
+  z-index: 11;
+}
 
-  h1,
-  p {
-    filter: drop-shadow(0 0 16px var(--primary-color));
+.PlayButton {
+  button {
+    background-color: #FED546;
+    border: none;
+    border-radius: 0.5rem;
+    font-weight: 700;
+    padding: 1rem;
+    cursor: pointer;
+  }
+  padding-bottom: 2.5rem;
+}
+
+.PlayButtonText {
+  padding: 1rem;
+  color: #003459;
+  font-size: 2.7rem;
+}
+
+.StartScreenHeader {
+    max-width: 80rem;
+    padding: 1.5rem 3rem 2.5rem;
+}
```

Comment:
Indentation is wrong - please consider using prettier in editor when saving code to resolve this